04_environment/index.tex

```diff
@@ -1 +1,400 @@
+
+\begin{figure}[!htb]
+    \centering
+    \includegraphics[width=1\textwidth]{04_environment/Images/Common Pool infrastructure outcome.PNG}
+    \caption{Demonstration of how common pool works in software}
+    \label{Images:Common Pool infrastructure outcome}
+\end{figure}
+
+The common pool threshold is a fixed value and known to islands. later, the functionality of making the Threshold unknown to the islands would be added. \\
+
+The common pool resources will be given to each island based on the severity of the damages.\\
+
+If the amount of resources in the common pool exceeds the total effect of disaster, then the leftover amount of resources in the common pool will be used for the next round.\\
+
+Since the goal of each island is to maximise its utility, the following dilemma shows up: islands can free ride by not contributing to the common pool and yet receiving the benefits after a disaster or if they are aware that they would not be depleted by the disaster - since they are far away from the eye of the storm - they might also not contribute to the common pool and thus the rest of the islands would be severely destroyed. \\
+
+The following example can briefly explain this dilemma:\\
+
+If the epicentre of disaster is known to the islands and happens to coincide with the location of Island 1, then Island 4 might not contribute at all to the CP since it will not be affected by the disaster. Moreover, Islands 3 and 5 that will not be severely hit might contribute a small amount to the CP or even zero amount (free riding case). Since the rest of the islands might contribute to the CP, then Islands 3 and 5 will be profited by the contribution of others.\\
+
+We wanted to highlight the importance of prevention and incentivise islands to contribute to the Common Pool, thus if the amount of resources in the common pool exceeds the threshold when the disaster happens, then the “power” of disaster towards the islands and common pool will be decreased. It makes sense in real life, as more preparation will result in less damage. The actual damage of disaster is decreased by half, but this can vary later.\\
+
+\subsection{Example distribution}
+This can be described by the examples in table X, which shows 4 possible scenario:\\ 
+
+\begin{itemize}
+    \item Contribution to common pool does not meet threshold and common pool cannot fully mitigate disaster’s damage
+    \item Contribution to common pool does not meet threshold but common pool can fully mitigate disaster’s damage
+    \item Contribution to common pool meets threshold but common pool cannot fully mitigate disaster’s damage
+    \item Contribution to common pool meets threshold and common pool can fully mitigate disaster’s damage
+\end{itemize}
+
+\begin{table}[h]
```

Comment:
the table simply doesn't fit the page -- adjust the column widths

Reply:
[h] -> [!htb]